packages/app/src/components/commons/PostItem.tsx

```diff
@@ -51,11 +51,11 @@ const PostItem: React.FC<PostItemProps> = ({ article, couldUpdate }) => {
   const { title, tags, lastUpdated, id } = article
   const articleTitle = shortTitle(title, 30)
   const date = lastUpdated && new Date(parseInt(lastUpdated) * 1000)
-
+  const publicationId = article.publication && article.publication.id
```

Comment:
Suggested simplification:
```const publicationId = article.publication?.id```

btw, all articles need to be tied to a publication so it should always be precent.